package.sh

```diff
@@ -26,7 +35,7 @@ rm -rf output/$APP_NAME/default/{restmap.conf,web.conf,bitwarden_event_logs_sett
 rm -rf output/$APP_NAME/README/bitwarden_event_logs_settings.conf.spec
 poetry run ucc-gen package --path output/$APP_NAME/ -o output/
 
-mv output/${APP_NAME}-${VERSION}.tar.gz output/bitwarden_event_logs.tar.gz
+mv "output/${APP_NAME}-${VERSION}.tar.gz" output/bitwarden_event_logs.tar.gz
 
 # Validate
-poetry run splunk-appinspect inspect --mode precert output/bitwarden_event_logs.tar.gz
+#poetry run splunk-appinspect inspect --mode precert output/bitwarden_event_logs.tar.gz
```

Comment:
Is the commenting out of this line intentional?

Reply:
Fixed in 23dc38e1